### DISCUSSION TITLE
Towny taxes dependant on PvP enabled or not

### DISCUSSION THREAD

**Discussion by destruct1** [TownyAdvanced/Towny#5750] [Q&A]
Hello there!

Is it possible to have whether or not a town pays taxes dependant on whether they have PvP enabled or not? 
For example, if a town has PvP enabled, the town pays no taxes and if a town has PvP disabled, the town has to pay taxes.

Thank you!!!

**Comment by LlmDl** [comment]
By tax are you referring to Upkeep?

**Reply by destruct1** [reply]
Upkeep, my apologies for not being clear.

**Reply by LlmDl** [reply]
A plugin can be made to do that using the API. It is not a feature built into Towny currently. You'd need someone to code you a plugin for it.

**Reply by destruct1** [reply]
Okay, thank you very much!
